cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -523,6 +526,30 @@
     },
     "system_details": {
       "properties": {
+        "container_images": {
+          "items": {
+            "properties": {
+              "description": {
+                "description": "A brief description of the container image",
+                "type": "string"
+              },
+              "recid": {
+                "description": "The internal ID of the container image, if it is another Open Data record",
+                "type": "string"
+              },
+              "registry": {
+                "description": "The type of registry the image can be found at",
+                "type": "string"
+              },
+              "uri": {
+                "description": "The URI for finding the container image",
```

Comment:
Again here:

- URI where the container image is located

However, we may perhaps be better off using a common term FQIN here to denote fully-qualified image name concept, i.e. the field would be called "fqin" instead of "uri" and the description would explain it as follows:

- Fully Qualified Image Name (FQIN) where the container image is located (repository/name:tag)

We can discuss live also with @katilp before merging

Reply:
P.S. BTW it the "fqin" field name risks looking a bit weird, we could use some longer name variant such as "fully_qualified_name", or just "name", and document that the name should be FQIN in the schema.